### PR TITLE
fix error into database

### PR DESCRIPTION
**Pull request by at-annguyen**
## Description
Change Database and restart model database into data seeder
## Reference Requirements
- Product Backlog ID: PBI-0
- Task ID: #0.2

## Migrations
YES 

## Impacted Areas in Application
List general components of the application that this PR will affect:

-

## SQL query and `explain`
List of all queries and attach the result of each explaining query (in text, screenshot...)

- Refer to [Using Explain with MySQL](https://viblo.asia/p/su-dung-explain-de-toi-uu-cau-lenh-mysql-BYjv44gmvxpV)

### CODE REVIEW

**Comment by at-anhngo** [comment]
LGTM

**Comment by at-duyho** [comment]
- add description, Impacted Areas in Application, SQL query and explain

File: database/seeds/PostsTableSeeder.php [hunk 1]
```diff
@@ -13,14 +13,115 @@ class PostsTableSeeder extends Seeder
...
-        Model::unguard();
-        $userId = DB::table('users')->pluck('id')->toArray();
-        $faker = Faker::create();
-        for ($i = 0; $i <= 15; $i++) {
-            factory(App\Model\Post::class, 1)->create([
-                'user_id' => $faker->randomElement($userId)
-            ]);
-        }
-        Model::reguard();
+        DB::table('posts')->insert([
```
Comment (at-duyho) [inline database/seeds/PostsTableSeeder.php]: - why don't you use factory?

File: database/migrations/2018_01_08_065800_change_posts_table.php [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+        Schema::table('posts', function (Blueprint $table) {
+            $table->dropColumn(['postable_id', 'postable_type']);
+        });
+        Schema::table('posts', function (Blueprint $table) {
+            $table->nullableMorphs('postable');
+            $table->string('type')->after('user_id');
+            $table->string('image')->after('content')->nullable();
+        });
+        Schema::table('posts', function (Blueprint $table) {
+            DB::statement('ALTER TABLE posts CHANGE COLUMN postable_id postable_id INT(11) NULL
```
Comment (at-duyho) [inline database/migrations/2018_01_08_065800_change_posts_table.php]: - why you change type column?

File: database/migrations/2018_01_08_065800_change_posts_table.php [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+        });
+        Schema::table('posts', function (Blueprint $table) {
+            $table->nullableMorphs('postable');
+            $table->string('type')->after('user_id');
+            $table->string('image')->after('content')->nullable();
+        });
+        Schema::table('posts', function (Blueprint $table) {
+            DB::statement('ALTER TABLE posts CHANGE COLUMN postable_id postable_id INT(11) NULL 
+            DEFAULT NULL AFTER user_id;');
+            DB::statement('ALTER TABLE posts CHANGE COLUMN postable_type postable_type VARCHAR(20) NULL
```
Comment (at-duyho) [inline database/migrations/2018_01_08_065800_change_posts_table.php]: - why you change type column?

File: database/migrations/2014_10_12_000000_create_users_table.php [hunk 4]
```diff
@@ -36,6 +36,8 @@ public function up()
      */
     public function down()
     {
+        DB::statement('SET FOREIGN_KEY_CHECKS=0');
```
Comment (at-duyho) [inline database/migrations/2014_10_12_000000_create_users_table.php]: - this doesn't necessary.

**Pull request closed without merging**